pkgs/facebook/buck2/pkg.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  - name: facebook/buck2@latest
```

Comment:
aqua doesn't support mutable versions such as `latest`.

Reply:
Hmm. If `cmdx s` generates this code, this is a bug of `aqua gr` command.
I'll take a look.

Reply:
18dff553f9aa73a1b9b0369f63edee9da3df3c44